packages/backend/src/queryBuilder.mock.ts

```diff
@@ -1036,7 +1036,7 @@ export const METRIC_QUERY_WITH_TABLE_CALCULATION_FILTER: CompiledMetricQuery = {
                         fieldId: 'calc3',
                     },
                     operator: FilterOperator.EQUALS,
-                    values: [],
+                    values: ['my value'],
```

Comment:
This unit test was wrong from the beginning. 
The table calculation id would not match with the filter target id (`table_calculation_{table name}`) causing the test to fall into this condition.
`if (!field) return '1=1';`